github/repos_collaborators.go

```diff
@@ -94,7 +94,7 @@ type RepositoryAddCollaboratorOptions struct {
 	Permission string `json:"permission,omitempty"`
 }
 
-// AddCollaborator adds the specified GitHub user as collaborator to the given repo.
+// AddCollaborator invites the specified GitHub user as a collaborator to the given repo.
```

Comment:
This is a good improvement, thanks. Let's make it even more explicit about what this does. This is based on wording in [GitHub API docs](https://developer.github.com/v3/repos/collaborators/#add-user-as-a-collaborator):

> To send an invitation to a collaborator rather than directly adding them,

So, how about:

```
// AddCollaborator sends an invitation to the specified GitHub user to become a collaborator to the given repo.
```